changelog

```diff
@@ -0,0 +1,7 @@
+changelog :
```

Comment:
This doesn't belong here. Either move it to the backends/platforms/psp2, or put on our Wiki after the merge